integration_tests/src/main/python/generate_expr_test.py

```diff
@@ -63,3 +68,33 @@ def test_explode_litarray(data_gen):
             lambda spark : four_op_df(spark, data_gen).select(f.col('a'), f.col('b'), f.col('c'), 
                 f.explode(array_lit)))
 
+#sort locally because of https://github.com/NVIDIA/spark-rapids/issues/84
+# After 3.1.0 is the min spark version we can drop this
+@ignore_order(local=True)
+def test_explode_split_string():
+    str_gen = StringGen('([ABC]{0,3}_?){0,7}').with_special_case('').with_special_pattern('.{0,10}')
+    assert_gpu_and_cpu_are_equal_collect(
+            lambda spark : two_col_df(spark, int_gen, str_gen).selectExpr('a', 'explode(split(b, "_"))'))
+    assert_gpu_and_cpu_are_equal_collect(
+            lambda spark : two_col_df(spark, int_gen, str_gen).selectExpr(
+                'a', 'explode(split(b, "_")) as c').selectExpr(
+                'a', 'explode(split(c, "A"))'))
+
+#sort locally because of https://github.com/NVIDIA/spark-rapids/issues/84
+# After 3.1.0 is the min spark version we can drop this
+@ignore_order(local=True)
+def test_explode_from_parquet(spark_tmp_path):
```

Comment:
What indication do we have that reading from parquet will cause different results compared to just doing the explode directly.  I personally would much rather see us cover all of the types that explode supports instead of covering a few things that might come before it in a plan.